app.php

```diff
@@ -407,6 +407,7 @@ public function load_admin_scripts($hook) {
 					'backbone',
 					'backbone.trackit',
 					'_ca_condition-groups',
+					'vaa_view_admin_as_script',
```

Comment:
Yes, that is a better idea. At some point I will move away from the default CPT admin screen, where there should be less chance of conflicting scripts.

This whitelist was made out of pure frustration that several plugins kept conflicting and not many developers wanted to do anything about it :(